FabNESO/tasks.py

```diff
@@ -21,10 +27,14 @@
     from types import SimpleNamespace
 
     fab = SimpleNamespace(task=lambda f: f, load_plugin_env_vars=lambda _: lambda f: f)
+    template = SimpleNamespace(
+        task=lambda f: f, load_plugin_env_vars=lambda _: lambda f: f
+    )
```

Comment:
I'm not sure if it did raise an error at some point (I'm not sure why I'd have thought to do the same thing otherwise...), but it doesn't seem to anymore, so I've taken out this part from the most recent version,